src/commands/releases.rs

```diff
@@ -443,21 +449,14 @@ fn execute_set_commits<'a>(
     let heads = if matches.is_present("auto") {
         let commits = find_heads(None, &repos, Some(config.get_cached_vcs_remote()))?;
         if commits.is_empty() {
-            let config = Config::current();
-
-            bail!(
-                "Could not determine any commits to be associated automatically.\n\
-                 Please provide commits explicitly using --commit | -c.\n\
-                 \n\
-                 HINT: Did you add the repo to your organization?\n\
-                 Configure it at {}/settings/{}/repos/",
-                config.get_base_url()?,
-                org
-            );
+            None
```

Comment:
Why did the logic change here? I thought `--auto` should keep working as before?

Reply:
If we cannot find commits associated with a repo-based integration, the new expected behaviour is to automatically push commits from local git tree. That's why I removed the bail.